lib/screen/account/persona_details_page.dart

```diff
@@ -155,6 +164,52 @@ class _PersonaDetailsPageState extends State<PersonaDetailsPage>
     );
   }
 
+  Widget _importInfo(BuildContext context) {
+    final theme = Theme.of(context);
+    return Padding(
+      padding: ResponsiveLayout.pageHorizontalEdgeInsets,
```

Comment:
Too many Nested Padding widget. Suggest to user `padding` and `margin` of Container. you can remove 2 Padding Widgets.